datafusion/substrait/src/logical_plan/consumer.rs

```diff
@@ -1297,6 +1297,32 @@ pub async fn from_substrait_rex(
                         outer_ref_columns,
                     })))
                 }
+                SubqueryType::SetPredicate(predicate) => {
+                    match predicate.predicate_op {
```

Comment:
It looks like we could use https://docs.rs/substrait/0.35.0/substrait/proto/expression/subquery/struct.SetPredicate.html#method.predicate_op to match on `PredicateOp` rather than a constant

So lke

```rust
match predicate.predicate_op() { 
  PredicateOp::Exists => ...
  other_type => ...
}
```

Reply:
Sure, I'll fix it.